.github/workflows/ci.yml

```diff
@@ -34,6 +34,7 @@ jobs:
     outputs:
       docker_image_digest: ${{ steps.docker_push.outputs.digest }}
       version: ${{ steps.docker_meta.outputs.version }}
+      sem_version_bool: ${{ steps.semantic.outputs.new_release_version }}
```

Comment:
The `new_release_version` var in this action is the new version created. `new_release_published` is a boolean though, maybe you mean this?
https://github.com/marketplace/actions/action-for-semantic-release#outputs